report-hazard/report-hazard.js

```diff
@@ -0,0 +1,96 @@
+const allPages = document.querySelectorAll('div.page');
+allPages[0].style.display = 'block';
+
+function navigateToPage(event) {
+    const pageId = location.hash ? location.hash : '#step1';
+    for (let page of allPages) {
+        if (pageId === '#' + page.id) {
+            page.style.display = 'block';
+        } else {
+            page.style.display = 'none';
+        }
+    }
+    return;
+}
+navigateToPage();
+
+//init handler for hash navigation
+window.addEventListener('hashchange', navigateToPage);
+
+
+class Report {
+    constructor(location, category, type, details, picture) {
+        this.location = location;
```

Comment:
use the following structure for this class

```
{
    "categoryId":"",
    "categoryOptionId":"",
    "location":{
        "lat":-87.8683,
        "lng":-147.2485,
        "address":"Northwest"
    },
    "comment":"",
    "images":["","",""]
}
```

this to match the API payload, if you need any other field let me know

Reply:
commit 7e8d30f08a070399dc745101fc75314d1fde0adf